packages/suite-desktop-core/webpack/core.webpack.config.ts

```diff
@@ -76,13 +76,25 @@ const config: webpack.Configuration = {
     target: 'electron-main',
     mode: isDev ? 'development' : 'production',
     devtool: 'source-map',
-    entry: ['app', 'preload', ...threads, 'winHelloChildProcess'].reduce(
+    entry: [
+        // NOTE: in DEV ONLY pick app with react dev tools installed
+        { app: isDev ? 'app-with-devtools' : 'app' },
```

Comment:
a better solution to this?

I wanted to not polut the "production" code with dev tools at all